frontend/app/templates/components/person-edit.hbs

```diff
@@ -84,15 +84,10 @@
             <table class="table-person">
               <tbody>
                 <tr>
-                  <td>
-                    <label class="label-date form-field--label" for="date_location">Geburtsdatum</label>
-                    <div id="date_location" class="birthdate_pikaday">
-                      <PikadayInput @onSelection={{action "setBirthdate"}} @value={{this.person.birthdate}} @useUTC={{true}} />
-                    </div>
-                  </td>
+                  <td>{{personChangeset.input label="Geburtsdatum" name="birthdate" renderComponent=(component "validation-date-input" input=this.person.birthdate) errorComponent=(component "validation-error-component")}}</td>
```

Comment:
This validation didn't work for me...

Reply:
It seems fine to me and also passed the test. What exactly didn't work?

Reply:
That one's on me... don't know why it didn't work. Works now though, sorry about that :sweat_smile: